CommonCompilerConfig.cmake

```diff
@@ -292,15 +271,6 @@ else()
 	set(cxx_flags_unoptimized ${cxx_flags_unoptimized} ${stdlib_common} -fdiagnostics-show-option)
 	set (internal_compiler_warning_flags ${internal_compiler_warning_flags} -Werror -Wextra -Wreturn-type -Wunused -Wno-unused-variable -Wno-unused-parameter -Wno-missing-field-initializers)
 
-	CHECK_CXX_COMPILER_FLAG(-m${BITNESS} compiler_supports_machine_option)
```

Comment:
I think we want to keep bitness and arm.  We eventually want to build for Rasberry Pi and such.

Reply:
wouldn't the `-marm` be auto recognized by cmake?

Reply:
> wouldn't the `-marm` be auto recognized by cmake?

good point.